python/eos/pyhf.py

```diff
@@ -0,0 +1,85 @@
+import eos
```

Comment:
This file is missing the typical boiler plate. See e.g. ``python/eos/analysis_file.py``.